src/main/java/me/totalfreedom/totalfreedommod/command/Command_playerverify.java

```diff
@@ -134,9 +134,36 @@ protected boolean run(CommandSender sender, Player playerSender, Command cmd, St
 
             case "settag":
             {
-                String tag = StringUtils.join(args, " ", 1, args.length);
-                target.setTag(tag);
-                msg("Your default tag is now: " + FUtil.colorize(target.getTag()));
+                final String inputTag = StringUtils.join(args, " ", 1, args.length);
+                final String outputTag = FUtil.colorize(StringUtils.replaceEachRepeatedly(StringUtils.strip(inputTag),
+                        new String[]
+                                {
+                                        "" + ChatColor.COLOR_CHAR, "&k"
+                                },
+                        new String[]
+                                {
+                                        "", ""
+                                })) + ChatColor.RESET;
+
```

Comment:
what is all of that  supposed to do lol

Reply:
checks for forbidden words because before a person could set their tag to admin or something

Reply:
^ with /pv but its blocked in all the tag commands so this was a loophole

Reply:
I mean what's all that StringUtils shit

Reply:
remove &k and strips &color codes from the 20 character limit